lms/templates/instructor/instructor_dashboard_2/data_download.html

```diff
@@ -50,7 +62,7 @@
   %endif
 
     <div class="request-response msg msg-confirm copy" id="report-request-response"></div>
-    <div class="request-response-error msg msg-warning copy" id="report-request-response-error"></div>
+    <div class="request-response-error msg msg-error copy" id="report-request-response-error"></div>
```

Comment:
@sarina This change makes sure that error messages for report generation tasks are actually displayed on the page:

The click handlers in [`data_download.coffee`](https://github.com/open-craft/edx-platform/blob/OC-791-answer-export/lms/static/coffee/src/instructor_dashboard/data_download.coffee) store error messages in the `request-response-error` div. Then they make that div visible via:

    $(".msg-error").css({"display":"block"})

So if the `request-response-error` div doesn't include the `msg-error` class it won't become visible in case of an error.

Another way to handle this would be to e.g. change the line above to

    $(".msg-warning").css({"display":"block"})

for all the click handlers. But this would require additional CSS changes because the error message ends up looking like this (unless that's what it's supposed to look like -- I don't know):

![error-warning-message](https://cloud.githubusercontent.com/assets/961441/9304592/10d1ddf2-44ea-11e5-89c4-26890d5aa4cd.png)

---

I do realize that the effect that this change has exceeds the scope of this PR. Please let me know how you would like to proceed.